docs/outputs.md

```diff
@@ -48,3 +48,105 @@ which will produce a ```png``` image per dump suitable for encoding into a movie
 ## Visualization software
 
 Both [ParaView](https://www.paraview.org/) and [VisIt](https://wci.llnl.gov/simulation/computer-codes/visit/) are capable of opening and visualizing Parthenon graphics dumps.  In both cases, the ```.xdmf``` files should be opened.  In ParaView, select the "XDMF Reader" when prompted.
+
+## Preparing outputs for ```yt```
+
+Parthenon HDF5 outputs can be read with the python visualization library
+[yt](https://yt-project.org/) as certain variables are named during when adding
+fields via ```StateDescriptor::AddField```. Variable names are added as a
+```std::vector<std::string>``` in the variable metadata. These labels are only
+used for output to HDF5. 4D variables are named with a list of names for each
+row while 3D variables are named with a single name.  For example, the
+following configurations are acceptable:
+
+```c++
+auto pkg = std::make_shared<StateDescriptor>("Hydro");
+
+/* ... */
+
+std::vector<std::string> cons_labels(nhydro);
+cons_labels[0]="Density";
+cons_labels[1]="MomentumDensity1";
+cons_labels[2]="MomentumDensity2";
+cons_labels[3]="MomentumDensity3";
+cons_labels[4]="TotalEnergyDensity";
+Metadata m({Metadata::Cell, Metadata::Independent, Metadata::FillGhost},
+           std::vector<int>({5}), cons_labels);
+pkg->AddField("cons", m);
+
+std::vector<std::string> density_labels(1);
+density_labels[0]="Density";
+m = Metadata({Metadata::Cell, Metadata::Derived}, std::vector<int>({1}), density_labels);
```

Comment:
Are these labels optional? Or are they now required?

Reply:
They're optional. If they're not specified, the HDF5 output will label the entire field by the field name, i.e. `cons` and `dens` here. I'll update the documentation.